lib/bots/versionbot.ts

```diff
@@ -92,20 +91,26 @@ interface ExternalCommand {
     };
 }
 
+enum ReviewState {
+    Approved = 0,
+    ChangesRequired
+};
```

Comment:
Sorry that was extremely unclear by me. I mean github comments, for example if somebody is requested for review, and they select `comment` (not `approve` or `request changes`) then github will remove them from the "to-review" list. I'm just wondering if this is something that you need to consider in the reviewer logic?